pom.xml

```diff
@@ -319,6 +319,9 @@ under the License.
                         <exclude>**/*.json</exclude>
                         <exclude>**/.editorconfig</exclude>
                         <exclude>**/config.gypi</exclude>
+                        <!-- ML -->
+                        <exclude>**/*.txt</exclude>
+                        <exclude>venv/**</exclude>
```

Comment:
As it's popular and common to use virtual environments (venv) with python, I've added the folder for the exclusion of the relevant files. Do you think it would make more sense to leave it as the responsibilities to the users using venv to deal with it on their own?

Reply:
Hmm.. it also assumes that users create `venv` folder in Nemo's root path..

I don't know well... but okay because it's very minor issue.